core/src/avm1.rs

```diff
@@ -895,26 +952,33 @@ impl<'gc> Avm1<'gc> {
     }
 
     fn action_enumerate(&mut self, context: &mut ActionContext<'_, 'gc, '_>) -> Result<(), Error> {
-        let name = self.pop()?;
-        let name = name.as_string()?;
+        let name_value = self.pop()?;
+        let name = name_value.as_string()?;
         self.push(Value::Null); // Sentinel that indicates end of enumeration
-        let ob = match self
-            .current_stack_frame()
+        self.current_stack_frame()
             .unwrap()
             .read()
             .resolve(name, self, context)
-        {
-            Value::Object(ob) => ob,
-            _ => {
-                log::error!("Cannot enumerate properties of {}", name);
-
-                return Ok(()); //TODO: This is NOT OK(()).
-            }
-        };
+            .and_then(
+                self,
+                context,
+                stack_continuation!(name_value: Value<'gc>, |avm, _context, object| {
+                    match object {
+                        Value::Object(ob) => {
+                            for k in ob.read().get_keys() {
+                                avm.push(Value::String(k));
```

Comment:
After #97 this can just be:
```suggestion
                                avm.push(k);
```